mmcv/runner/hooks/logger/pavi.py

```diff
@@ -61,7 +65,13 @@ def before_run(self, runner):
             self.init_kwargs = dict()
         self.init_kwargs['task'] = self.run_name
         self.init_kwargs['model'] = runner._model_name
-
+        if runner.meta is not None and 'config_dict' in runner.meta:
+            meta = runner.meta['config_dict'].copy()
+            meta.setdefault('max_iter', runner.max_iters)
```

Comment:
What's the purpose of this line?

Reply:
`max_.*iter` will be parsed as the maximum iteration to show the training progress bar in pavi.

Reply:
We can leave a comment here.